frontend/src/app/[organisation]/bemanning/api/updateHours/route.ts

```diff
@@ -16,13 +17,19 @@ export async function POST(
     searchParams.get("selectedWeek") || undefined,
   );
 
+  const body: StaffingWriteModel = {
+    type: bookingType,
+    consultantId: Number(consultantID),
+    engagementId: Number(engagementID),
+    year: selectedWeek?.year ?? 0,
+    week: selectedWeek?.weekNumber ?? 0,
+    hours: Number(hours),
+  };
+
   const staffing =
-    (await postWithToken(
-      `${orgUrlKey}/consultants/staffing/new?Hours=${hours}&Type=${bookingType}&ConsultantID=${consultantID}&EngagementID=${engagementID}&${
-        selectedWeek
-          ? `Year=${selectedWeek.year}&Week=${selectedWeek.weekNumber}`
-          : ""
-      }`,
+    (await putWithToken<never, StaffingWriteModel>(
```

Comment:
Is there a good alternative to using <never> here?

Reply:
Se comment above

Reply:
Typescript has many types of "some kind of object" base types. They're the same thing under the hood, but give some signals to the developer: 

- `never` - I'm not going to use this object, so I don't care about the type
- `unknown` - I can't know what type of object this is
- `Object` - I'm just going to use basic Object-functions, so I don't need a typed object. 
- `any` - I'm transitioning from Javascript to Typescript, I'm very lazy or have no idea what I'm doing.